src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,9 +6,63 @@ cloud: true
 docs_area: deploy
 ---
 
-This page provides important recommendations for CockroachDB {{ site.data.products.cloud }} production deployments.
+Before deploying CockroachDB {{ site.data.products.cloud }} in production, it is important to understand the Shared Responsibility Model that delineates the responsibilities of Cockroach Labs and the customer in managing CockroachDB {{ site.data.products.cloud }} clusters. 
 
-## Follow the SQL Best Practices
+Under the Shared Responsibility Model, Cockroach Labs is responsible for the following tasks: 
+
+- Cluster and cloud service availability and reliability
```

Comment:
Some of the items in this and the following list are long enough that I think each item should end in a period.